pkg/kapp/app/apps.go

```diff
@@ -83,7 +86,15 @@ func (a Apps) list(additionalLabels map[string]string, nsName string) ([]App, er
 	}
 
 	for _, app := range apps.Items {
-		recordedApp := &RecordedApp{app.Name, app.Namespace, app.ObjectMeta.CreationTimestamp.Time, a.coreClient,
+		name := app.Name
+		isMigrated := false
+
+		if _, ok := app.Labels[KappIsConfigmapMigratedLabelKey]; ok {
```

Comment:
Should this be an annotation instead?
I think we wanna use labels when we are going to use the label as a selector while querying those resources, (example `kapp.k14s.io/is-app` used as selector when user runs `kapp ls`)
However, since we are comparing after getting the resource, I think it should be an annotation.

Reply:
Thanks @100mik ! Good call, changed to using annotations. Looks like this is our first instance of an annotation, so I kept the label name the same just as an annotation instead

Reply:
Sounds perfect! Thanks